@types/nextjs-routes.d.ts

```diff
@@ -100,7 +100,7 @@ declare module "next/link" {
   export interface LinkProps
     extends Omit<NextLinkProps, "href" | "locale">,
       AnchorHTMLAttributes<HTMLAnchorElement> {
-    href: Route | StaticRoute | Omit<Route, "pathname">
+    href: Route | StaticRoute | Omit<Route, "pathname"> | DynamicRoute;
```

Comment:
Note: This was needed for using the Next Link Component in the Navbar. That implementation was decided upon via [this post](https://stackoverflow.com/questions/32106513/material-ui-menu-using-routes). Bc we could use the MUI Button Component but limits our custom styling options.

Reply:
Spoke offline, this file should not be modified and the changes can be made in Navbar component